### PR TITLE
chore: deps bump

### PR DESCRIPTION
**Pull request by bogdan-bashev**
https://veriff.atlassian.net/browse/DEVXP-2331

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -16,13 +16,13 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x, 19.x]
+        node-version: [ 20.x, 22.x]
```
Comment (ckvrff) [inline .github/workflows/test.yml:19]: We still have 18.x as part of Veriff/base-images (i.e. this version of node is used in prod), should we remove 18.x or no then? https://github.com/Veriff/base-images?tab=readme-ov-file#node-lts

**Review by ckvrff — APPROVED** [review]
Looks fine at a glance.

File: .github/workflows/test.yml (hunk 1 above)
Comment (bogdan-bashev) [reply .github/workflows/test.yml:19]: i think with 18 something need downgrade a little (again) will check
most pain are creata-react-app (2y old) and airbnb was also discontinued a year ago...

Comment (ckvrff) [reply .github/workflows/test.yml:19]: Yeah, forget it. Maybe it'll give a small push to update to 20 then 👍

**Review by eritikass — APPROVED** [review]

**Review by madisp — APPROVED** [review]

**Pull request merged**